lucet-runtime/lucet-runtime-internals/src/region/mmap.rs

```diff
@@ -78,6 +79,7 @@ impl RegionInternal for MmapRegion {
         &self,
         module: Arc<dyn Module>,
         embed_ctx: CtxMap,
+        heap_memory_size: usize,
```

Comment:
In the context of a struct called `Limits`, it's clear that `heap_memory_size` is a limit. But just as a bare argument to this function, it's less clear. Maybe call this `heap_memory_size_limit` like you do with the builder method?